api/tools/src/main/resources/bigquery/es_person.sql

```diff
@@ -0,0 +1,56 @@
+SELECT
```

Comment:
Is this sql going to be a template for all domains? If so should you parameterized anything that references a domain? or this can be done later as this code matures.

Reply:
I'm planning to throw the whole mess into here in a subsequent PR as one giant query. I think each domain will have slightly different values being mapped into these structs.